<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageGUI</class>
 <widget class="QDialog" name="ImageGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TerraME Observer : Image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filename prefix :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editPrefix">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Show the prefix of the images files.</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(235, 235, 235);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Path:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editPath">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Location where images are stored.</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(235, 235, 235);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImageGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
